src/components/OpenFileDialog/OpenFileDialog.js

```diff
@@ -124,7 +125,10 @@ export const OpenFileDialog = ({
         }
 
         // for ER 2.38 only show line list ER types
-        if (type === AO_TYPE_EVENT_REPORT) {
+        if (
+            type === AO_TYPE_EVENT_REPORT ||
+            type === AO_TYPE_EVENT_VISUALIZATION
+        ) {
```

Comment:
This part has been removed here:
https://github.com/dhis2/analytics/pull/1126/files#diff-785f63826da7b159716f77c9f8a6ddee63e24884dfb3ae542445f66b890b0a76L127
Because now we rely only on the `visType` for the filtering, not the AO type.

This will result in a conflict.